.github/workflows/test-action.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: Test ansible-test-gh-action
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  tests:
+    name: Basic tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ansible-core-version: stable-2.12
```

Comment:
This needs to pass yamllint (and I should probably integrate it to run automatically before accepting this).

Reply:
Implemented in d904ec757b4e8bc3dba10960b6f4aeefd0f6a9d3.